modules/codegen-cli/src/smithy4s/codegen/cli/CodegenCommand.scala

```diff
@@ -116,7 +117,7 @@ object CodegenCommand {
             defaultDependencies ++ dependencies.getOrElse(List.empty)
           }
           CodegenArgs(
-            specsArgs,
+            specsArgs.map(PathRef(_)),
```

Comment:
**thought:** / issue: I thought we'd keep `CodegenArgs` as-is, given that `PathRef`'s hashing is used by the build tool, and not the underlying codegen. i.e. you'd unwrap PathRef in `codegenArgsIso`

Reply:
Agreed

Reply:
Fair point, I've restored the public API for `CodegenArgs`. `PathRef` is only used for serialization now

Reply:
@majk-p, did you push your latest commit ? I'm not seeing it.

Reply:
I didn't :facepalm: Pushed now